manifest/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+authors = ["o2sh <ossama-hjaji@live.fr>"]
+name = "manifest"
```

Comment:
Looks like [manifest is already taken](https://crates.io/crates/manifest). Perhaps rename this to `onefetch-manifest`? If this sub-crate can't be published to crates.io, I don't think that onefetch itself can be published to crates.io. Also, with multiple crates in the same repo, should this project be converted to a [workspace](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html)?

Reply:
>Looks like [manifest is already taken](https://crates.io/crates/manifest). Perhaps rename this to onefetch-manifest?

🤦 Good catch, thanks @spenserblack 

> should this project be converted to a [workspace](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html)?

done, maybe we should do the same thing  to the `image_backends` module 🤔

Reply:
> maybe we should do the same thing ... to the `image_backends` module

Thanks makes sense to me :+1:
onefetch probably has a lot of usage that can go into separate crates that others may want to use (kind of related: https://github.com/o2sh/onefetch/pull/851#discussion_r1016683417). I think this `onefetch-manifest` crate is a really cool tool that could evolve into its own project (I'm thinking of it like a tokei for your manifests).